metadata_backend/api/handlers.py

```diff
@@ -500,7 +500,7 @@ def _check_patch_folder(self, patch_ops: Any) -> None:
                     raise web.HTTPBadRequest(reason=reason)
                 pass
             else:
-                if all(i not in op["path"] for i in _required_paths + _arrays):
+                if all(i not in op["path"] for i in _required_paths + _arrays + ["/doiInfo"]):
```

Comment:
why not add `/doiInfo` to `_arrays` ?

Reply:
added it to arrays, but it also complicates the other tests